### ISSUE TITLE
The question about INF network parameters 

### ISSUE THREAD

**Issue by NewGod** [carpedm20/ENAS-pytorch#6]
When I  was running your code, I found that some network parameters into INF. Is there any suggestion  to solve this problem. Thanks.
By the way, I was curiosity about the test ppl after training.

**Comment by bkj** [comment]
Try reducing the learning rate -- that might help the inf problem
